publish-service/src/main/java/com/ericsson/eiffel/remrem/publish/config/SecurityConfig.java

```diff
@@ -52,25 +58,49 @@ public class SecurityConfig extends WebSecurityConfigurerAdapter {
 
     @Value("${activedirectory.userSearchFilter}")
     private String userSearchFilter;
-    
+
     @Value("${activedirectory.rootDn}")
     private String rootDn;
 
-    @Override
-    protected void configure(AuthenticationManagerBuilder auth) throws Exception {
+    @Value("${activedirectory.connectionTimeOut:#{0}}")
+    private Integer ldapTimeOut;
```

Comment:
What's advantage of using Integer over int? Finally, a method setting the timeout to rabbitmq channel accepts int. Using int, instead of Integer, simplifies its handling as it cannot be null.